pkg/util/commons.go

```diff
@@ -154,6 +154,18 @@ func IsPhoneNumber(str string) bool {
 	return re.MatchString(str)
 }
 
+// FormatUSPhoneNumber formats the given phone number to: +1 (XXX) XXX-XXXX
+func FormatUSPhoneNumber(phoneNumber string) string {
+	if phoneNumber == "" {
```

Comment:
Instead of repeating code here can't we just call `FormatPhone` function and just do the last formatting of the returned value?